debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -1048,4 +1049,32 @@ public static int validateTopicName(Configuration config, Field field, Validatio
         }
         return 0;
     }
+
+    public static int notContainEmptyElements(Configuration config, Field field, Field.ValidationOutput problems) {
+
+        if (!config.hasKey(field)) {
+            return 0;
+        }
+
+        List<String> values = config.getList(field);
+        if (values.contains(EMPTY_STRING)) {
+            problems.accept(field, values, "not permitted empty string");
+            return 1;
+        }
+        return 0;
+    }
+
+    public static int notContainSpaceInAnyElements(Configuration config, Field field, Field.ValidationOutput problems) {
```

Comment:
I have one issue with this one. As it might sound strange people (esp. SQL Server users) are using spaces in colmn/table names. Would it be possible to add support for the spaces in the for of escaped space or something like that?

Reply:
@jpechane Can you please clarify 

> Would it be possible to add support for the spaces in the for of escaped space or something like that?

Since is it possible to have spaces in column/table names, this check here is not necessary. The only space that can cause some issue can be "change . column with space" just for the space around the dot. I can solve it with a trim after split. WDYT?